modules/aggregation-layers/src/screen-grid-layer/screen-grid-cell-layer.ts

```diff
@@ -152,8 +153,10 @@ export default class ScreenGridCellLayer<DataT = any, ExtraPropsT extends {} = {
           positions: new Float32Array([0, 0, 0, 1, 0, 0, 1, 1, 0, 0, 1, 0])
         }
       }),
-      // @ts-expect-error TODO v9 API not as dynamic
-      isInstanced: true
+      bufferLayout: [
```

Comment:
Can you not do `bufferLayout: this.getAttributeManager()!.getBufferLayouts()`?

Reply:
Ah that's better yes — thanks!